manifests/charts/istio-control/istio-discovery/files/kube-gateway.yaml

```diff
@@ -47,9 +47,18 @@ spec:
   template:
     metadata:
       annotations:
+        {{- $ambientRedirectionAnnotation := strdict}}
+        {{- if eq (env "PILOT_ENABLE_AMBIENT_CONTROLLERS" "false") "true" }}
+            {{- if eq .Values.global.ambient.kubeGatewayRedirection true}}
```

Comment:
I don't know we need a global value for this, we could just allow users to opt in explicitly by setting the `ambient.istio.io/redirection`?

```
if user has ambient.istio.io/redirection { use it } 
else { set ambient.istio.io/redirection=disabled }
```

?

Reply:
+1 - would be nice we sort this out automatically for the user without configuration.

Reply:
Makes sense, will update it when I'm back.